js/pages/characterizations/components/feature-analyses/feature-analysis-view-edit.html

```diff
@@ -27,7 +27,7 @@
         </div>
     </div>
 
-    <div data-bind="css: classes('content'), visible: !$component.loading()">
+    <div data-bind="css: {'content': true, 'fe-design-disabled': !$component.canEdit()}, visible: !$component.loading()">
```

Comment:
Take a look, `classes('content')` isn't the same as `'content': true`, this change breaks the panel style